guides/doc-Managing_Hosts/topics/host_management_and_monitoring_using_red_hat_web_console.adoc

```diff
@@ -1,7 +1,7 @@
 [[host_management_and_monitoring_using_red_hat_web_console]]
-= Host Management and Monitoring Using Red Hat Web Console
+= Host Management and Monitoring Using {cockpit}
 
-Red Hat web console is an interactive web interface that you can use to perform actions and monitor Red Hat Enterprise Linux hosts. You can enable a remote-execution feature to integrate {Project} with Red Hat web console. When you install Red Hat web console on a host that you manage with {Project}, you can view the Red Hat web console dashboards of that host from within the {Project} web UI. You can also use the features that are integrated with Red Hat web console, for example, Red Hat Image Builder.
+{cockpit} is an interactive web interface that you can use to perform actions and monitor Red Hat Enterprise Linux hosts. You can enable a remote-execution feature to integrate {Project} with {cockpit}. When you install {cockpit} on a host that you manage with {Project}, you can view the {cockpit} dashboards of that host from within the {Project} web UI. You can also use the features that are integrated with {cockpit}, for example, {lorax-composer}.
```

Comment:
Wow this is confusing: `{cockpit}` resolves to `{Cockpit}` with capital letter. We have Foreman=Foreman already. Would you mind fixing this until there are just few of them there? The attribute was already merged into master...

Reply:
Oh no, I did it again. I didn't commit the attributes file. It's not already in master. I was silly.

Reply:
Are you sure? https://github.com/theforeman/foreman-documentation/blob/master/guides/common/attributes.adoc#L53

What I am asking you here, if you don't mind, is actually fixing this attribute altogether. :-)

Reply:
Btw git blame shows who commited it half an year ago: https://github.com/theforeman/foreman-documentation/commit/7c5fdfbddf705ecab1961b0690f94259a454018a

Hahahaahah

Reply:
I was just about to type that! 
TBH I was really surprised that these attributes weren't implemented. 
I almost remember doing it .... :)

Reply:
so s{cockpit}/{Cockpit} if I understand correctly @lzap ?